prime-router/metadata/schemas/NM/nm-covid-19-csv.schema

```diff
@@ -155,30 +193,86 @@ elements:
   - name: first_test
     csvFields:
       - name: FirstTest
+        format: $alt
+    altValues:
+      - code: Y
+        display: Y
+      - code: N
+        display: N
+      - code: UNK
+        display: U
 
   - name: employed_in_healthcare
     csvFields:
       - name: EmpHealthcare
+        format: $alt
+    altValues:
+      - code: Y
+        display: Y
+      - code: N
+        display: N
+      - code: UNK
+        display: U
 
   - name: symptomatic_for_disease
     csvFields:
       - name: Symptomatic
+        format: $alt
+    altValues:
+      - code: Y
+        display: Y
+      - code: N
+        display: N
+      - code: UNK
+        display: U
 
   - name: hospitalized
     csvFields:
       - name: Hospitalized
+        format: $alt
+    altValues:
+      - code: Y
+        display: Y
+      - code: N
+        display: N
+      - code: UNK
+        display: U
 
   - name: icu
     csvFields:
       - name: ICU
+        format: $alt
+    altValues:
+      - code: Y
+        display: Y
+      - code: N
+        display: N
+      - code: UNK
+        display: U
 
   - name: resident_congregate_setting
     csvFields:
       - name: CongSetting
+        format: $alt
+    altValues:
+      - code: Y
+        display: Y
+      - code: N
+        display: N
+      - code: UNK
+        display: U
 
   - name: pregnant
     csvFields:
       - name: Pregnant
+        format: $alt
+    altValues:
+      - code: Y
+        display: Y
+      - code: N
+        display: N
+      - code: UNK
+        display: U
```

Comment:
This is nuts.   We _really_ need to implement our plan to genericize tables/valuesets/altvalues etc.